components/add_users_to_group_modal/add_users_to_group_modal.tsx

```diff
@@ -71,12 +71,24 @@ const AddUsersToGroupModal = (props: Props) => {
         } else {
             goBack();
         }
-    };
+    }, []);
```

Comment:
I miss here the `goBack` dependency

Reply:
and maybe, but I don't think is going to change, the `props.actions.addUsersToGroup`